simtools/db_handler.py

```diff
@@ -198,6 +198,22 @@ def exportFileDB(self, dbName, dest, fileName):
         file = self._getFileMongoDB(dbName, fileName)
         self._writeFileFromMongoToDisk(dbName, dest, file)
 
+    def exportFilesDB(self, dbName, dest, fileNames):
```

Comment:
I don't think we need to create this function. The two reasons I see are the following,
1. The user can easily just put this for loop in their application, it doesn't make a huge difference to the user to have it in the API instead.
2. Even if we do want to have this functionality in the API, it's much better to have a single function that deals both with a string or a list of strings then having two separate functions.

Reply:
Ok, I would go for your second point. In this case we should also merge insertFileToDB and insertFilesToDB in simtools/db_handler.py to match the style here.

Reply:
Notice that `insertFilesToDB` doesn't include the protection in the case it's not a list (perhaps it's worth changing it as well, not sure). Also it is divided into two functions which is unnecessary in the export case (might be worth changing the insert case as well). What I would do is change the export method do be something like the usual

```
if not isinstance(file, list):
    filesToExport = [file]
else:
    filesToExport = file
for fileNow in filesToExport:
    self._logger.debug(f"Getting {fileName} and writing it to {dest}")
    file = self._getFileMongoDB(dbName, fileName)
    self._writeFileFromMongoToDisk(dbName, dest, file)
```